models/user.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/python3
+"""Module for the User class"""
+
+from models.base_model import BaseModel
+
+
+class User(BaseModel):
+    """
+    class User inherited form class BaseModel
+    Handles users information
+    """
+    email = ""
+    password = ""
+    first_name = ""
+    last_name = ""
```

Comment:
Do we need to have an init function in each of the subclasses?

I was thinking it would have something like
class User(BaseModel):
    ...
   def __init__(self, *args, **kwargs):
          # Documentation
          super().__init__(*args, **kwargs)